Document/PostManager.php

```diff
@@ -89,4 +142,23 @@ public function getPublicationDateQueryParts($date, $step, $alias = 'p')
             ),
         );
     }
+
+    /**
+     * @param string $collection
+     *
+     * @return array
+     */
+    protected function getPublicationCollectionQueryParts($collection)
```

Comment:
Same reason here, but I made it final now, because there is no reason to extend it.